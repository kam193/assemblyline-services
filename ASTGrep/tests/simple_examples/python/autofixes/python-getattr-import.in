eval("getattr")(eval("__import__")("os"), "open")
eval(b"getattr")(eval('__import__')("subprocess"), '''check_output''')
getattr(obj, "sth", "default")
