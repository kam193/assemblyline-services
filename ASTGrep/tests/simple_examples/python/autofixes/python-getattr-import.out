os.open
subprocess.check_output
getattr(obj, "sth", "default")
